doc/AUTHORS

```diff
@@ -1,5 +1,5 @@
 Team:
-https://suricata-ids.org/about/team/
+https://suricata.io/about/team/
 
 All contributors:
 https://www.ohloh.net/p/suricata-engine/contributors/summary
```

Comment:
Ohloh appears to be dead? I know its not part of this PR but falls into URL cleanup.

Reply:
True, Shivani had mentioned that a while ago, as well... this works: https://www.openhub.net/p/suricata-engine/contributors/summary
Would it be interesting if I replaced ohloh with that one?